### PR TITLE
add a SUBTITLE to sensible places and other small fixes

### PR DESCRIPTION
**Pull request by szaimen**

### CODE REVIEW

File: addons/locate_mirror.sh [hunk 1]
```diff
@@ -50,5 +51,6 @@ else
     then
         sudo mv sources.list /etc/apt/
     fi
+    msg_box "The apt-mirror was successfully changed."
```
Comment (enoch85) [inline addons/locate_mirror.sh:54]: Move this within the if argument please.

**Comment by enoch85** [comment]
Side note, 

Not when we made such big changes in the scripts. Please test 19.0.2 and choose to install everything. Just to make sure we didn't break something. Please don't forget to run without internet  as well to test the static_ip script properly as well.

I would if I had time, but I don't right now.

**Comment by enoch85** [comment]
> I would if I had time, but I don't right now.

I will take care of it. :+1:

**Comment by enoch85** [comment]
All is good! :D

**Comment by enoch85** [comment]
@szaimen Please fix travis.

Other than that I think it looks OK to merge.

**Comment by szaimen** [comment]
> @szaimen Please fix travis.

done

**Pull request merged**